mkosi

```diff
@@ -2852,7 +2916,13 @@ def make_tar(args: CommandLineArguments,
 
     with complete_step('Creating archive'):
         f: BinaryIO = cast(BinaryIO, tempfile.NamedTemporaryFile(dir=os.path.dirname(args.output), prefix=".mkosi-"))
-        run(["tar", "-C", root, "-c", "-J", "--xattrs", "--xattrs-include=*", "."],
+# some distributions defaults to bsdtar(libarchive) which uses POSIX argument list so let's keep a separate list
+        if shutil.which('bsdtar'):
```

Comment:
Maybe make this a bit less intrusive, prefer tar over bsdtar but if tar is not found, check if bsdtar exists and use it if it does.

Reply:
@DaanDeMeyer in case of libarchive it does provide tar as a symlink to bsdtar.

There are two options.
1. Parse tar --version output for bsdtar string
2. Check of tar is not a soft/hard link to badtar

Reply:
I think we can make this a distribution check for now (just if mandriva => bsdtar). We can always revisit this if we need bsdtar for something else as well.

Reply:
fixed, now it checks for bsdtar and Distrobution.openmandriva